.github/workflows/publish.yml

```diff
@@ -113,3 +113,39 @@ jobs:
         run: echo '//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}' > .npmrc
       - run: npm publish
         working-directory: "oso-js-node-${{ github.event.inputs.version }}"
+  rust:
+    name: Publish Rust crates to Crates.io
+    runs-on: ubuntu-18.04
+    env:
+      CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-release-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Does this work as a cache key?

Reply:
why wouldn't it?

Reply:
(I.e. does the package step use the same files as `cargo build --release`?

Reply:
Sometimes it has to rebuild a bunch of things anyway

Reply:
Actually I think cargo publish always does a full rebuild anyway so it doesn't matter. I'll just drop the cacheing for these.